README.md

```diff
@@ -97,6 +97,35 @@ class MyApp < Sinatra::Base
 end
 ```
 
+### If using Plain Ruby
+
+Rollbar isn't dependent on Rack or Rails for most of its functionality. In a regular script, assuming you've
+installed the rollbar gem:
+
+ 1. Require rollbar
+ 2. Configure rollbar
+ 3. Send Rollbar data
+
+```ruby
+require 'rollbar'
+
+Rollbar.configure do |config|
+  config.access_token = "SERVER_POST_ACCESS_TOKEN"
```

Comment:
This should be "POST_SERVER_ITEM_ACCESS_TOKEN" to be consistent with the rest of this file (and work with the replacements that happen in our logged-in site)